src/encoding.rs

```diff
@@ -89,9 +90,19 @@ impl EncodingOverride {
     }
 }
 
+#[cfg(feature = "query_encoding")]
+impl Debug for EncodingOverride {
```

Comment:
Can't derive Debug in this case because encoding field has type Option<EncodingRef> and EncodingRef trait isn't Debug